app/styles/pages/events/sessions.scss

```diff
@@ -0,0 +1,17 @@
+.left-float{
```

Comment:
Sorry, but I am unable to find from where should I use these pre-existing semantic ui classes with which name they are present.